### PR TITLE
chore: fix function names in comment

### PR DESCRIPTION
**Pull request by looklose**
## Description

 fix function names in comment

## Todos

- [ ] Tests
- [x] Documentation
- [ ] Release note

## Release Note

```release-note
 fix function names in comment
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

**Comment by looklose** [comment]
friendly ping~

File: apiserver/cmd/apiserver/server/server.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func logrusLevel() logrus.Level {
 	return logrus.ErrorLevel
 }
 
-// NewCommandStartMaster provides a CLI handler for 'start master' command
+// NewCommandStartCalicoServer provides a CLI handler for 'start master' command
```
Comment (nelljerram) [inline apiserver/cmd/apiserver/server/server.go:56]: Better just to delete this comment IMO.  The `'start master'` part is wrong, and it's really not adding any value.

File: apiserver/pkg/storage/calico/resource.go [hunk 2]
```diff
@@ -96,7 +96,7 @@ func CreateClientFromConfig() clientv3.Interface {
 	return c
 }
 
-// Versioned returns the versioned associated with this interface
+// Versioner returns the versioned associated with this interface
```
Comment (nelljerram) [inline apiserver/pkg/storage/calico/resource.go:99]: Need to change the second "versioned" to "versioner".

**Review by nelljerram — CHANGES_REQUESTED** [review]

File: apiserver/cmd/apiserver/server/server.go (hunk 1 above)
Comment (looklose) [reply apiserver/cmd/apiserver/server/server.go]: Thanks! Modified

File: apiserver/pkg/storage/calico/resource.go (hunk 2 above)
Comment (looklose) [reply apiserver/pkg/storage/calico/resource.go]: Thanks! Modified

**Review by nelljerram — APPROVED** [review]

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**